
spring.jpa.hibernate.ddl-auto=update
#spring.datasource.url=jdbc:mysql://34.95.177.62/cerebro
#spring.datasource.url=jdbc:postgresql://ec2-52-0-155-79.compute-1.amazonaws.com:5432/d32g52qdf6i1si
#spring.datasource.username=qjrjldiefeuddl
#spring.datasource.password=
spring.datasource.url=jdbc:mysql://localhost:3306/cerebro
spring.datasource.username=root
spring.datasource.password=


server.port=8081
server.error.include-stacktrace=never


#spring.cloud.gcp.sql.database-name=cerebro
#spring.cloud.gcp.sql.instance-connection-name=cerebro-281813:southamerica-east1:cerebro123

# So app starts despite "table already exists" errors.
spring.datasource.continue-on-error=true
# Enforces database initialization
spring.datasource.initialization-mode=always
#spring.cloud.gcp.project-id=cerebro-281813
#spring.cloud.gcp.credentials.location=file:/src/main/resources/cerebro-281813-33180cf98e8a.json